src/pages/instances/InstanceSearchFilter.tsx

```diff
@@ -64,59 +60,56 @@ const InstanceSearchFilter: FC<Props> = ({ instances }) => {
       .map((chip) => chip.value as LxdInstanceStatus);
     const filteredTypes = searchData
       .filter((chip) => chip.lead === "type")
-      .map((chip) => (chip.value === "VM" ? "virtual-machine" : "container"));
+      .map((chip) => chip.value);
 
     const profileQueries = searchData
       .filter((chip) => chip.lead === "profile")
       .map((chip) => chip.value);
 
-    const url = new URL(window.location.origin + window.location.pathname);
-
+    const currentWindowSearchParams = new URLSearchParams(
+      window.location.search,
+    );
+    const newSearchParams = new URLSearchParams();
     filteredQueries.forEach((query) => {
-      url.searchParams.append("queries", query);
+      newSearchParams.append("q", query);
     });
 
     filteredStatuses.forEach((status) => {
-      url.searchParams.append("statuses", status);
+      newSearchParams.append("status", status);
     });
 
     filteredTypes.forEach((type) => {
-      url.searchParams.append("types", type);
+      newSearchParams.append("type", type);
     });
     profileQueries.forEach((profile) => {
-      url.searchParams.append("profileQueries", profile);
+      newSearchParams.append("profile", profile);
     });
-    if (url.href !== window.location.href) {
-      setSearchParams(url.searchParams);
+    if (newSearchParams.toString() !== currentWindowSearchParams.toString()) {
+      setSearchParams(newSearchParams);
     }
   };
 
   const getChips = (): SearchAndFilterChip[] => {
     const searchParams = new URLSearchParams(window.location.search);
     const statusChipUrl = [
```

Comment:
I just realised that this now looks pretty similar to the items that we have below, inlined with the `newSearchData` array. I wonder if this should be inlined there too, instead of having, it alone, its own `const` declaration?